.gitignore

```diff
@@ -1,8 +1,3 @@
-libjuliampi.so
-libjuliampi.dylib
 *.cov
-*.dSYM
 *.mem
-*.so
 Manifest.toml
-LocalPreferences.toml
```

Comment:
Why not git ignore that? It can still occur during development?

Reply:
Fine, I kept it :stuck_out_tongue: